.github/workflows/api.yml

```diff
@@ -58,10 +76,14 @@ jobs:
           npm run build -w packages/client
           echo "$(date --utc --iso-8601=seconds) web3.storage upload test" > ./upload-test-small
           ./packages/w3/bin.js put ./upload-test-small --api https://api-staging.web3.storage --token ${{ secrets.STAGING_WEB3_TOKEN }}
-  release:
-    name: Release
+  
+  deploy-production:
+    name: Deploy Production
+    if: (github.event_name == 'push' && github.ref == 'refs/heads/main' && needs.changelog.outputs.releases_created) || inputs.force_release
     runs-on: ubuntu-latest
-    needs: test
+    needs:
+      - test
+      - changelog
```

Comment:
release please step below should go away right?

Reply:
ah yes

Reply:
Fixed